sdk/include/ddk/d3dkmthk.h

```diff
@@ -5649,15 +5649,13 @@ typedef _Check_return_ NTSTATUS (APIENTRY *PFND3DKMT_CANCELPRESENTS)(_In_ D3DKMT
 
 #endif
 
-#ifdef __REACTOS__
-#if (DXGKDDI_INTERFACE_VERSION >= DXGKDDI_INTERFACE_VERSION_WDDM2_2)
+#if !defined(__REACTOS__) || (DXGKDDI_INTERFACE_VERSION >= DXGKDDI_INTERFACE_VERSION_WDDM2_2)
```

Comment:
Please, double-check in case you meant `3_0`.

Reply:
I think the idea of the original code was to keep it unchanged the most. So, the if DXGKDDI thing and its endif with comment would have to stay as they were.

The ifdef REACTOS -> ifndef REACTOS on the contrary would be the real fix.
But then, seeing that nothing in our code base was really affected by the inclusion of these definitions, I wonder whether any if(n)def REACTOS for these is actually necessary.

Cc @DarkFire01

Reply:
> the if DXGKDDI thing and its endif with comment

No, they are ROS-specific too.

Reply:
IMO #ifdef __REACTOS__ should only be used seperate to other definition  Checks and only to distinct between ROS and 3rdparty code.